parbase/FairParRootFileIo.cxx

```diff
@@ -147,33 +151,31 @@ Bool_t FairParRootFileIo::open(const Text_t* fname, Option_t* option, const Text
 
 Bool_t FairParRootFileIo::open(const TList* fnamelist, Option_t* option, const Text_t* ftitle, Int_t compress)
 {
-    TDatime currentDate;
-    TString newParFileName = "";
-    TFile* newParFile = 0;
-
+    char fname[100];
```

Comment:
Have you considered using `std::array<char, 100> fname`? see [ES.27](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es27-use-stdarray-or-stack_array-for-arrays-on-the-stack) and [SL.con.1](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#slcon1-prefer-using-stl-array-or-vector-instead-of-a-c-array)